src/org/jitsi/impl/neomedia/MediaStreamImpl.java

```diff
@@ -3678,10 +3688,47 @@ public boolean isEndOfFrame(byte[] buf, int off, int len)
     }
 
     /**
-    * {@inheritDoc}
-    */
+     * {@inheritDoc}
+     * </p>
+     * This is absolutely terrible, but we need a RawPacket and the method is
+     * used from RTPTranslator, which doesn't work with RawPacket.
+     */
     public boolean isKeyFrame(byte[] buf, int off, int len)
     {
+        return isKeyFrame(new RawPacket(buf, off, len));
```

Comment:
instead of creating a new object, can we make this the main method and pass pkt.getBuffer() etc from the other method?